homeassistant/components/alexa/capabilities.py

```diff
@@ -994,6 +999,29 @@ def capability_resources(self):
             self._resource.add_mode(f"{cover.ATTR_POSITION}.custom", ["Custom"])
             return self._resource.serialize_capability_resources()
 
+        if self.instance == f"{light.DOMAIN}.{light.ATTR_EFFECT}":
+            self._resource = AlexaModeResource(["Effect"], False)
+
+            effect_list = self.entity.attributes.get(light.ATTR_EFFECT_LIST, 0)
+            if effect_list:
+                for effect in effect_list:
+                    self._resource.add_mode(f"{effect}", [f"{effect}"])
+
+            return self._resource.serialize_capability_resources()
+
+        if self.instance == f"{light.DOMAIN}.{light.ATTR_PROFILE}":
+            self._resource = AlexaModeResource(["Profile"], False)
+
+            await light.Profiles.load_profiles(self.hass)
```

Comment:
This is not an exposed API of the light integration and should be considered internal. We should not expose this.

Reply:
Is there alternative way to get a list of effects @balloob ?

Reply:
There is not right now. I am also not happy with profiles to begin with. It's a hacked on feature from the early days of Home Assistant. I suggest we limit this PR to effects for now.

Reply:
OK no problem. Will update later today to strip out the profiles stuff and add tests.